lib/public/view.js

```diff
@@ -35,6 +34,7 @@ import { TagDetailsPage } from './views/Tags/Details/TagDetailsPage.js';
 import { EosReportCreationPage } from './views/EosReport/create/EosReportCreationPage.js';
 import { StatisticsPage } from './views/Statistics/StatisticsPage.js';
 import { LhcPeriodsOverviewPage } from './views/lhcPeriods/Overview/LhcPeriodsOverviewPage.js';
+import { HomeOverviewPage } from './views/Home/Overview/index.js';
```

Comment:
I think you should simply name this HomePage

Reply:
renamed